src/py/flwr/superexec/exec_servicer_test.py

```diff
@@ -44,7 +44,7 @@ def test_start_run() -> None:
     request.fab_file = b"test"
 
     # Create a instance of FlowerServiceServicer
-    servicer = ExecServicer(executor=executor)
+    servicer = ExecServicer(executor=executor, config=None)
```

Comment:
This is not necessary now that we have `set_config`

Reply:
In the servicer we still need it, because it is there that we will call the `set_config` method